addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/AbstractHomematicGateway.java

```diff
@@ -645,12 +631,15 @@ public void eventReceived(HmDatapointInfo dpInfo, Object newValue) {
         String className = newValue == null ? "Unknown" : newValue.getClass().getSimpleName();
         logger.debug("Received new ({}) value '{}' for '{}' from gateway with id '{}'", className, newValue, dpInfo,
                 id);
-        lastEventTime = System.currentTimeMillis();
 
         if (echoEvents.remove(dpInfo)) {
             logger.debug("Echo event detected, ignoring '{}'", dpInfo);
         } else {
             try {
+                if (newValue != null && connectionTrackerThread != null && dpInfo.isPong() && id.equals(newValue)) {
```

Comment:
The `newValue != null` is not needed, it's already covered with `id.equals(newValue)`: because nothing is equal to `null`